cluster/common.sh

```diff
@@ -423,7 +435,9 @@ EOF
   fi
 
   if [[ -n "${GEN_ETCD_CA_CERT}" && -n "${GEN_ETCD_CA_KEY}" ]]; then
+    # shellcheck disable=SC2154
```

Comment:
please don't disable without leaving a comment explaining why we chose to disable a check, so we may fix it in the future or understand that it is actually correct (rare).

Reply:
good point!